lib/services/appmanager-service.ts

```diff
@@ -99,5 +143,27 @@ class AppManagerService implements IAppManagerService {
 			}
 		}).future<void>()();
 	}
+
+	private findGroups(identityStrings:string[]): IFuture<string[]> {
+		return ((): string[] => {
+			let availableGroups = this.$server.tam.getGroups().wait();
+
+			if (!availableGroups.length) {
+				this.$errors.failWithoutHelp("No available groups found.");
+			}
+
+			return _.map(identityStrings, identityStr => {
+				let group = helpers.findByNameOrIndex(identityStr, availableGroups, (group) => group.Name);
+
+				if (!group) {
+					this.$errors.failWithoutHelp ("Could not find group named '%s' or was not given a valid index.%sTo see available groups run $ appbuilder appmanager groups",
```

Comment:
Cannot find group that matches the provided \<Group ID\>: '%s'. To list the available groups, run `$ appbuilder appmanager groups`